src/vm/frames.h

```diff
@@ -2535,6 +2535,10 @@ class GCFrame : public Frame
 
     // Keep as last entry in class
     DEFINE_VTABLE_GETTER_AND_DTOR(GCFrame)
+
+#if !defined(FEATURE_PAL) && !defined(DACCESS_COMPILE) && !defined(CROSSGEN_COMPILE)
```

Comment:
Nit: `FEATURE_PAL` can be removed from this ifdef if the macro above is changed to just `DEFINE_VTABLE_GETTER`.

Reply:
I didn't want to do that, I wanted all frame kinds on Unix to have destructor defined in an uniform manner at the same place and be explicit for the Windows GCFrame.

Reply:
ok

Reply:
Note that `GCFrame::Pop` does more stuff than `Frame::Pop`. May want to keep calling `GCFrame::Pop` from the `GC_PROTECT` macro on Unix so that keep doing that extra stuff on Unix.

Reply:
Hmm, then I guess that I'll actually follow your original advice and rather make the GCFrame destructor be the same for Windows and Linux instead of making the GCPROTECT_END different for Windows and Linux.